test/combineReducers.spec.js

```diff
@@ -0,0 +1,78 @@
+import { combineReducers } from '../src';
+import { getModel } from '../src/loop';
+import { Map } from 'immutable';
+
+const reducers = {
+  counter: (state = 0, action = {}) => state + 1,
+  doubler: (state, action = {}) => state ? state + state : 1,
+  fibonacci: (state = 1, action = {}) => action.previous ? action.previous + state : state
+};
+
+describe('combineReducers', () => {
+  it('works with one argument and returns correctly working reducer', () => {
```

Comment:
so one thing that i've found in working with jest is that making it statements apply to as few `expect`s as possible makes debugging test failures a lot more pleasant - otherwise it can get hairy pretty quickly to figure out which of the assertions is failing. just something to keep in mind as we write tests or attempt to debug test failures

Reply:
yea i agree. the combineReducers file and the index file though i just took the existing tests and converted them to jest. maybe at a later point we can break up the tests more.